gateway/src/apicast/policy/logging/logging.lua

```diff
@@ -77,6 +77,7 @@ local function get_request_context(context)
   }
 
   ctx.service = context.service or {}
+  ctx.service.host = context.host
```

Comment:
I think this modifies the `host` attr for the service meaning that every request for that service will be affected.

Reply:
This happens on log_by_lua block, so it only modifies when the request finishes, so nothing is affected no?